src/django/api/constants.py

```diff
@@ -186,3 +186,7 @@ class NumberOfWorkersRanges:
         'min': 10001,
         'label': 'More than 10000'
     }]
+
+
+class ErrorMessages:
+    GEOCODING_NO_RESULTS = "The address you submitted can not be geocoded."
```

Comment:
Do you think we can name it `GEOCODED_NO_RESULTS` to have a similar naming to the status `FacilityListItem.GEOCODED_NO_RESULTS`?

Reply:
Done.